<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QNewRepositoryDialog</class>
 <widget class="QDialog" name="QNewRepositoryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Clone / Add / Create Repository</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabCloneRepository">
      <attribute name="title">
       <string>Clone Repository</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBoxCloneRepository">
         <property name="title">
          <string>Bookmarks</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBoxCloneRepositoryBookmark">
            <property name="text">
             <string>&amp;Bookmark this repository</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelCloneRepositoryBookmark">
            <property name="text">
             <string>&amp;Name:</string>
            </property>
            <property name="buddy">
             <cstring>lineEditCloneRepositoryBookmark</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditCloneRepositoryBookmark"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelCloneRepositorySourceURL">
         <property name="text">
          <string>Source &amp;URL:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditCloneRepositorySourceURL</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelCloneRepositoryDestinationPath">
         <property name="text">
          <string>Destination &amp;Path:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditCloneRepositoryDestinationPath</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditCloneRepositoryDestinationPath"/>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="toolButtonCloneRepositoryBrowse">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEditCloneRepositorySourceURL"/>
       </item>
       <item row="2" column="0" colspan="3">
        <spacer name="verticalSpacerCloneRepository">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAddWorkingCopy">
      <attribute name="title">
       <string>Add Working Copy</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="labelAddWorkingCopyPath">
         <property name="text">
          <string>Working Copy &amp;Path:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditCloneRepositoryDestinationPath</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditAddWorkingCopyPath"/>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="toolButtonAddWorkingCopyBrowse">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBoxAddWorkingCopy">
         <property name="title">
          <string>Bookmarks</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBoxAddWorkingCopyBookmark">
            <property name="text">
             <string>&amp;Bookmark this repository</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelAddWorkingCopyBookmark">
            <property name="text">
             <string>&amp;Name:</string>
            </property>
            <property name="buddy">
             <cstring>lineEditCloneRepositoryBookmark</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditAddWorkingCopyBookmark"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <spacer name="verticalSpacerAddWorkingCopy">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>355</width>
           <height>98</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCreateNewRepository">
      <attribute name="title">
       <string>Create New Repository</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="labelCreateNewRepositoryPath">
         <property name="text">
          <string>Destination &amp;Path:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditCloneRepositoryDestinationPath</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditCreateNewRepositoryPath"/>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="toolButtonCreateNewRepositoryBrowse">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <spacer name="verticalSpacerCreateNewRepository">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>355</width>
           <height>98</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBoxCreateNewRepository">
         <property name="title">
          <string>Bookmarks</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBoxCreateNewRepositoryBookmark">
            <property name="text">
             <string>&amp;Bookmark this repository</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelCreateNewRepositoryBookmark">
            <property name="text">
             <string>&amp;Name:</string>
            </property>
            <property name="buddy">
             <cstring>lineEditCloneRepositoryBookmark</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditCreateNewRepositoryBookmark"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditCloneRepositorySourceURL</tabstop>
  <tabstop>lineEditCloneRepositoryDestinationPath</tabstop>
  <tabstop>toolButtonCloneRepositoryBrowse</tabstop>
  <tabstop>checkBoxCloneRepositoryBookmark</tabstop>
  <tabstop>lineEditCloneRepositoryBookmark</tabstop>
  <tabstop>lineEditAddWorkingCopyPath</tabstop>
  <tabstop>toolButtonAddWorkingCopyBrowse</tabstop>
  <tabstop>checkBoxAddWorkingCopyBookmark</tabstop>
  <tabstop>lineEditAddWorkingCopyBookmark</tabstop>
  <tabstop>lineEditCreateNewRepositoryPath</tabstop>
  <tabstop>toolButtonCreateNewRepositoryBrowse</tabstop>
  <tabstop>checkBoxCreateNewRepositoryBookmark</tabstop>
  <tabstop>lineEditCreateNewRepositoryBookmark</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QNewRepositoryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QNewRepositoryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
